src/aggregators/directcr.jl

```diff
@@ -56,11 +56,9 @@ function DirectCRJumpAggregation(nj::Int, njt::T, et::T, crs::Vector{T}, sr::T,
     minrate = 2.0^minexponent
     ratetogroup = rate -> priortogid(rate, minexponent)
 
-    # construct an empty initial priority table -- we'll overwrite this in init anyways...
-    rt = PriorityTable{T,Int,Int,typeof(ratetogroup)}(minrate, 2*minrate,
-                                                        Vector{PriorityGroup{T,Vector{Int}}}(),
-                                                        Vector{T}(), zero(T),
-                                                        Vector{Tuple{Int,Int}}(), ratetogroup)
+    # construct an empty initial priority table -- we'll reset this in init
+    num_jumps = length(crs)
+    rt = PriorityTable(ratetogroup, zeros(T, num_jumps), minrate, 2*minrate)
```

Comment:
Will this make the first bucket really big in memory (length num_jumps), and hold that memory forever?

Reply:
(since you don't reset the memory used by the table in `reset!`?)

Reply:
Yes, it will. And that's bad. I'll change that.